rules-tests/Removing/Rector/ClassMethod/ArgumentRemoverRector/config/configured_rule.php

```diff
@@ -7,16 +7,17 @@
 use Rector\Removing\ValueObject\ArgumentRemover;
 use Rector\Tests\Removing\Rector\ClassMethod\ArgumentRemoverRector\Source\Persister;
 use Rector\Tests\Removing\Rector\ClassMethod\ArgumentRemoverRector\Source\RemoveInTheMiddle;
-use Symfony\Component\Yaml\Yaml;
+use Rector\Tests\Removing\Rector\ClassMethod\ArgumentRemoverRector\Source\Yaml;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig
         ->ruleWithConfiguration(ArgumentRemoverRector::class, [
-            new ArgumentRemover(Persister::class, 'getSelectJoinColumnSQL', 4, null), new ArgumentRemover(
+            new ArgumentRemover(Persister::class, 'getSelectJoinColumnSQL', 4, null),
+            new ArgumentRemover(
                 Yaml::class,
                 'parse',
                 1,
-                ['Symfony\Component\Yaml\Yaml::PARSE_KEYS_AS_STRINGS', 'hey', 55, 5.5]
+                [Yaml::REMOVED_CONSTANT, 'hey', 55, 5.5]
```

Comment:
should keep using string `Symfony\Component\Yaml\Yaml::PARSE_KEYS_AS_STRINGS` reference, `Symfony\Component\Yaml\Yaml` no need to be exists to make transformation happen.

Reply:
Adjusted.